src/CoaddPsf.cc

```diff
@@ -152,9 +152,18 @@ CoaddPsf::CoaddPsf(
 {
     afw::table::SchemaMapper mapper(catalog.getSchema());
     mapper.addMinimalSchema(afw::table::ExposureTable::makeMinimalSchema(), true);
+
+    // copy the field "goodpix", if available, since computeAveragePosition will use it, if available
+    try {
+        afw::table::Key<int> goodPixKey = catalog.getSchema()["goodpix"]; // auto does not work
+        mapper.addMapping(goodPixKey, true);
+    } catch (pex::exceptions::NotFoundError &) {}
```

Comment:
A pretty trivial point, but: you've used "if available" twice one one line above, plus a further twice in the commit message, plus one instance of "if it was available". None of them are actually wrong, but all that repetition makes things pretty hard to read! Could you spend a few seconds rephrasing for clarity?